monitor/utils/message.js

```diff
@@ -66,10 +66,32 @@ const unclaimedHomeToForeignRequests = () => {
   }
 }
 
+const manuallyProcessedAMBHomeToForeignRequests = () => {
+  if (MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST) {
+    const allowanceList = readAccessListFile(MONITOR_HOME_TO_FOREIGN_ALLOWANCE_LIST)
+    return e => {
+      const { sender, executor, decodedDataType } = normalizeAMBMessage(e)
+      return (
+        decodedDataType.manualLane ||
+        (!allowanceList.includes(sender.toLowerCase()) && !allowanceList.includes(executor))
+      )
+    }
+  } else if (MONITOR_HOME_TO_FOREIGN_BLOCK_LIST) {
+    const blockList = readAccessListFile(MONITOR_HOME_TO_FOREIGN_BLOCK_LIST)
+    return e => {
+      const { sender, executor, decodedDataType } = normalizeAMBMessage(e)
+      return decodedDataType.manualLane || blockList.includes(sender.toLowerCase()) || blockList.includes(executor)
```

Comment:
The monitor needs to consider a message as forwarded to the manual lane regardless to `decodedDataType.manualLane` value: if either an account sender or a contract receiver are in the block list, the oracles don't trust it -- so, they don't take into consideration the value of the `decodedDataType.manualLane` bit.

Reply:
Yes, that is the desired behaviour. That's why we have two OR conditions here. If either manualLane bit is set, or some participating address is blocked, the message is considered as forwarded to the manual lane.

Reply:
Ok. I see now. Thanks.